### PR TITLE
create a service account preset, use it for test-infra

### PR DESCRIPTION
**Pull request by BenTheElder**
TODO: switch ALL the jobs next, but first let's try this on test-infra

/area jobs

### CODE REVIEW

File: prow/config.yaml [hunk 1]
```diff
@@ -94,6 +94,21 @@ tide:
     kubernetes/website: squash
   target_url: https://prow.k8s.io/tide.html
 
+presets:
+- labels:
+    preset: service-account
```
Comment (krzyzacy) [inline prow/config.yaml]: this is a list, so I guess `- preset: service-account`

Comment (BenTheElder) [reply prow/config.yaml]: other way, it's a map, the jobs have labels written as lists which is wrong. fixing

**Comment by BenTheElder** [comment]
/hold
bad rebase :(

**Pull request merged**